pkg/cidata/cidata.TEMPLATE.d/boot.sh

```diff
@@ -64,5 +59,9 @@ if [ -d "${LIMA_CIDATA_MNT}"/provision.user ]; then
 	done
 fi
 
+# Signal that provisioning is done. The instance-id in the meta-data file changes on every boot,
+# so any copy from a previous boot cycle will have different content.
+cp "${LIMA_CIDATA_MNT}"/meta-data /etc/lima-boot-done
```

Comment:
Can we use `/run/lima` rather than `/etc`?

Reply:
If you need persistence it should be `/var/lib/lima`

Reply:
No, `/run` is perfect.

I want the opposite of persistence, which is why I copy the `meta-data` file, which I know has different content on every restart. Because we have no way to "reset" the ready markers before cloud-init allows ssh access, so the requirements checks can move past them before they have been reset. Having a different content for the marker on every boot solves this.

Updated